app/views/govuk_publishing_components/components/_inverse_header.html.erb

```diff
@@ -2,9 +2,13 @@
   full_width ||= false
   header_class = full_width ? "gem-c-inverse-header--full-width" : ""
 %>
+<%
```

Comment:
No need to stop and start the rails sections here, they can just be part of the same thing.